app/src/app/present-round/present-round.component.ts

```diff
@@ -45,9 +48,69 @@ export class PresentRoundComponent implements OnInit {
         return val as boolean
       })
     )
+    this.markedDecreeAsSent = this.db.object("delegateRounds/" + this.delegateId + "/" + this.roundId + "/markedDecreeAsSent").valueChanges().pipe(
+      map(val => {
+        return val as boolean
+      })
+    )
+    this.db.list("landsraad/questions").snapshotChanges().pipe(
+      tap(
+        questionsSnapshots => {
+          this.availableQuestions = questionsSnapshots.map(qSnapshot => {
+            const question: any = qSnapshot.payload.val()
+            const qKey = qSnapshot.key
+            return {
+              ...question,
+              id: qKey
+            }
+          })
+        }
+      )
+    ).subscribe(ref => this.controlledVotingRights = this.db.list("landsraad/votingRights", ref => ref.orderByChild("controlledBy").equalTo(this.delegateId)).snapshotChanges().pipe(
+      map(
+        snapshots => {
+          return snapshots.map(snapshot => {
+            let value = snapshot.payload.val()
+            let existingQuestions = this.availableQuestions.filter(q => {
+              // check if there is existing question for this round, delegate, house 
+              // and not of the type "Bill of Particulars" (because that one is extra)
+              // we will use the same path if it exists
+              return q["byDelegateId"] === this.delegateId && q["byHouse"] === value["name"] && q["roundId"] === this.roundId && q["decretType"] !== "Bill of Particulars"
```

Comment:
Remove Bill of particulars for now to simplify this PR

Reply:
Ah ok will do and then add it when I add the "Bill of Particulars" in the 3rd PR. Because we will need it, so we do not "overwrite" that path by mistake.

Reply:
Removed.